torch_xla/csrc/tensor_impl.cpp

```diff
@@ -115,9 +119,8 @@ at::IntArrayRef XLATensorImpl::sizes_custom() const {
 }
 
 c10::SymIntArrayRef XLATensorImpl::sym_sizes_custom() const {
```

Comment:
Wondering what the "custom" in `sym_sizes_custom()` implies

Reply:
`custom` means that a given `TensorImpl` can implement `sym_sizes` in whichever way they see fit. Note, `sym_sizes_custom` is a virtual method whereas `sym_sizes()` isn't. `sym_sizes()` checks if a given tensor impl implements `CustomSizes` policy and if so calls a virtual sym_sizes. This is kinda convoluted and was done to make sure we don't pay a virtual call penalty on the fast path: cpu and cuda tensors don't need the custom_sym_sizes implementation.

Reply:
Thanks